flutter_app/lib/drone_information.dart

```diff
@@ -0,0 +1,80 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+import 'package:imacs/mavlink_communication.dart';
+
+import 'data_field_widget.dart';
+
+/// Widget to arrange multiple data widgets in tabular form.
+///
+/// This widget  displays all the data fetched from
+/// Mission Planner MAVLink. It automatically sets the data in a two
```

Comment:
This isn't really true, I don't think we fetch all the data MavLink makes available and we're definetly not getting it from Mission Planner.